docs/src/components/ColorSwatch/ColorSwatch.component.tsx

```diff
@@ -0,0 +1,105 @@
+// REACT
+import React, { useState } from 'react';
+// VENDOR
+import styled from 'styled-components';
+import { ColorHelpers } from '../ColorHelpers';
+import { Pill } from '../Pill'
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { Typography } from '../../../../src/Typography';
+// THEME
+import { colors } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchProps {
+  colorHexCode: string;
+  colorName: string;
+}
+
+
+// STYLES
+const StyledColorSwatch = styled.div`
+  border: 0.5rem solid ${props => props.highContrastBorderColor}};
+  display: inline-block;
+  margin: 0 2rem 2rem 0;
+  width: 17rem;
+
+  .color-swatch-pill-container {
+    background: ${props => props.colorHexCode };
+    cursor: pointer;
+    padding: 7rem 1rem 1rem 1.5rem;
+  }
+
+  .color-swatch-text-container {
+    border-top: 0.05rem solid ${props => props.highContrastBorderColor};
+    padding: 1rem;
+  }
+
+  .color-swatch-text-container p {
+    margin: 0;
+    padding: 0.25rem 0 0.25rem 0.5rem;
+  }
+
+  .color-swatch-text-container p:hover {
+    background: ${colors.ash.light};
+    border-radius: 1rem;
+    cursor: pointer;
+    text-decoration: underline;
+  }
+
+  .temporary-copy-confirmation {
+    font-size: 1rem;
```

Comment:
All of this can come from the Typography component that probably didn't exist when you started this work. @bjohnsonrmn I know you've got other things on your plate and this should have been called out earlier, so would you like me to refactor this with Typography, or would you like to work on that?

Reply:
@sharkparty I used `Typography` for some other elements, but this one seemed complex enough to deserve a class. I can re-work it, but are you saying every CSS attribute can be worked into `Typography` or only some of them?

Reply:
The only ones that can't would be position, top, right, border, & bg. A few of those will be changing when we build out a composable base component. I can definitely be convinced this is better. Are those the only props and usage that's outside of Typo...y?

Reply:
We talked this over in person and decided to leave as is 👍